.github/workflows/release-lib.yaml

```diff
@@ -38,6 +38,8 @@ jobs:
             target: x86_64-unknown-linux-gnu
           - os: ubuntu-22.04
             target: aarch64-unknown-linux-gnu
+          - os: ubuntu-20.04
+            target: x86_64-unknown-linux-gnu
```

Comment:
Could you please change the existing version of ubunt-22.04 to 20.04 instead of adding a new object?

My understanding is that binaries built on Ubuntu 20.04 can be used on Ubuntu 22.04, etc., so there is no need to do multiple builds. (For example, the mdBook repository once had a version downgrade from Ubuntu 22.04 to 20.04)

Reply:
OK, I updated both the x86_64 and aarch64 to ubuntu-20.04